machines-endpoints/pkg/machines-endpoints/main.go

```diff
@@ -319,9 +319,20 @@ func getBootServers(members []member) []net.IP {
 }
 
 func localHTTPClient() *http.Client {
-	transport := http.DefaultTransport.(*http.Transport).Clone()
-	transport.Proxy = nil
-	transport.ForceAttemptHTTP2 = false
+	// Most of the following values are copied from http.DefaultTransport to workaround a proxy issue.
+	// See: https://github.com/golang/go/issues/25793
+	transport := &http.Transport{
+		Proxy: nil,
+		DialContext: (&net.Dialer{
+			Timeout:   30 * time.Second,
+			KeepAlive: 30 * time.Second,
+			DualStack: true,
+		}).DialContext,
+		MaxIdleConns:          100,
```

Comment:
where is `ForceAttemptHTTP2:     true,`

Reply:
@ymmt2005 
Originally it was implicitly false by not specifying the value.
Should I specify the value explicitly?
https://github.com/cybozu/neco-containers/blame/0c7bd7da4f451206f55330a571c4b82b40eff393/machines-endpoints/pkg/machines-endpoints/main.go

Reply:
sorry, don't understand.
I saw this commit and there was the line...
https://github.com/cybozu/neco-containers/commit/e823b5e9b8ab00f4a25f119a3f9a59040958c2fc

Reply:
ah, I saw a wrong file!! sorry!